service/forward_service.cc

```diff
@@ -333,7 +333,7 @@ static shared_ptr<cql3::selection::selection> mock_selection(
     ) {
         auto name_as_expression = [] (const sstring& name) -> cql3::expr::expression {
             return cql3::expr::unresolved_identifier {
-                make_shared<cql3::column_identifier_raw>(name, false)
+                make_shared<cql3::column_identifier_raw>(name, true)
```

Comment:
It'd be more readable to name this boolean value
```suggestion
                constexpr bool keep_case = true
                make_shared<cql3::column_identifier_raw>(name, keep_case)
```

Reply:
It feels a bit overkill to me, but I was very curious how the `Commit suggestion` button works, so it's done now.
It would be nice if c++ had python's syntax `, keep_case=true)`

Reply:
AFAIK named parameters proposal for C++ has been discussed nearly as long as template's constraints/concepts, but proved to be too difficult to implement (to many corner cases).

Reply:
Ah the suggestion wasn't a valid C++ code, and the CI failed, fixed manually.